doc/sphinx/user/run-aspect/visualizing-results/index.md

```diff
@@ -29,15 +28,20 @@ being widely distributed:
     time step that then contains a reference to the individual files that
     together make up the output of a single time step. Unfortunately, there
     doesn't appear to be a standard for these master records; however,
-    both ParaView and Visit have defined a format that each of these programs
+    both ParaView and VisIt have defined a format that each of these programs
     understand and that requires placing a file with ending `.pvtu` or
     `.visit` into the same directory as the output files from each processor.
-    {ref}`4.2][] gives an example of what can be found in the output
+    {ref}`4.2`[] gives an example of what can be found in the output
     directory.
 
-<div class="center">
-
-</div>
+:::{note}
+You can select other formats for output than VTU, see the run-time parameters in Section
+A.165. However, none of the numerous formats currently implemented in deal.II other than
```

Comment:
This needs to become a reference of some sort. The section numbers change dynamically. How did you usually do things to remind everyone that something has to change?